### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by JuniorJPDJ**

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -16,5 +16,9 @@
       "groupName": "OS Packages"
     }
   ],
-  "forkProcessing": "enabled"
+  "forkProcessing": "enabled",
+  "ignoreDeps": [
+    "ghcr.io/openscopeproject/ziprofs",
+    "ghcr.io/juniorjpdj/ziprofs"
```
Comment (qu1ck) [inline renovate.json]: Why is this here?

Comment (JuniorJPDJ) [reply renovate.json]: Renovate is opening PRs for docker-compose, which is just an example and would be bumped after every commit (as image rebuild is triggered by commit).
Example:
https://github.com/JuniorJPDJ/ZipROFS/pull/4
https://github.com/JuniorJPDJ/ZipROFS/pull/2

If you mean why both images - it's easier to me to maintain it like this. If you want I can only keep your one and keep mine in separate commit in my fork.

Comment (qu1ck) [reply renovate.json]: Yeah, either use the image from my repo or if you are using yours then keep dependency exception in your repo too.

Comment (JuniorJPDJ) [reply renovate.json]: Done!
